### PR TITLE
Correct Borghans model, apply parameter scaling for fixed parameters

### PR DESCRIPTION
**Pull request by yannikschaelte**

### CODE REVIEW

File: tests/benchmark-models/benchmark_models.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ Boehm_JProteomeRes2014:
   note: benchmark collection reference ignores factor 1/2
 
 Borghans_BiophysChem1997:
-  llh: -3471.740659965799
-  note: benchmark collection reference value does not match, but model outputs do. maybe due to D2D data normalization
+  llh: 175.90130682476047
+  note: benchmark collection reference value does not match, but model outputs do. due to D2D data normalization
```
Comment (dweindl) [inline tests/benchmark-models/benchmark_models.yaml]:
```suggestion
  note: matches benchmark collection, except for sign
```
right?

Comment (yannikschaelte) [reply tests/benchmark-models/benchmark_models.yaml]: benchmark says -192, so we need to add the log10-normal correction sum(log10(sim)), then yes we have minus the benchmark value ...

Comment (dweindl) [reply tests/benchmark-models/benchmark_models.yaml]: okay, please update accordingly

Comment (yannikschaelte) [reply tests/benchmark-models/benchmark_models.yaml]: done

**Pull request merged**